frame/society/src/lib.rs

```diff
@@ -1304,7 +1334,7 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
 			Err(_) => Err(Error::<T, I>::NotMember)?,
 			Ok(i) => {
 				members.remove(i);
-				T::MembershipChanged::change_members_sorted(&[], &[m.clone()], &members[..]);
+				T::MembershipChanged::change_members_sorted(&[], &[m.clone()], &members.to_vec()[..]);
```

Comment:
```suggestion
				T::MembershipChanged::change_members_sorted(&[], &[m.clone()], &members);
```

I think this will work as well since `BoundedVec` implements `AsRef<T>`